### DISCUSSION TITLE
complete Support CycloneDX 1.5

### DISCUSSION THREAD

**Discussion by briannv** [CycloneDX/cyclonedx-python-lib#422] [General]
Hi team,

Do you have the ETA to support version 1.5?

Thanks,
Brian

**Comment by jkowalleck** [comment]
nope. see also: https://github.com/CycloneDX/cyclonedx-python-lib/labels/schema%201.5

this project is a community effort.
if you have an urgent need for CDX 1.5, then feel free to open an issue and make it clear that you are interested in providing that feature via a pull request.
